src/angularComponent/ngcOmniboxSuggestionsDirective.js

```diff
@@ -28,7 +28,36 @@ export default function ngcOmniboxSuggestionsDirective(ngcModifySuggestionsTempl
     controller() {},
     controllerAs: 'suggestions',
     compile(tElement) {
-      tElement.html(ngcModifySuggestionsTemplate(tElement[0]));
+      const doc = $document[0];
+      const element = tElement[0];
+      const tmpl = ngcModifySuggestionsTemplate(element);
+      const isElemDirective = element.localName === 'ngc-omnibox-suggestions';
+
+      // Wrap all of the element contents so we can put an ng-if on it. The wrapper should use
+      // all the attributes passed in by the app-maker except for the directive since that would
+      // cause an infinite loop
+      const wrapper = doc.createElement(isElemDirective ? 'div' : element.localName);
```

Comment:
`element.localName` has questionable cross-browser support: https://developer.mozilla.org/en-US/docs/Web/API/Element/localName

Does TAGNAME not work here?